src/ripple/core/impl/JobQueue.cpp

```diff
@@ -420,19 +420,27 @@ JobQueue::processTask (int instance)
             }
             type = job.getType();
             JobTypeData& data(getJobTypeData(type));
-            JLOG(m_journal.trace()) << "Doing " << data.name () << " job";
-            auto const us = date::ceil<microseconds>(
+            JLOG(m_journal.trace()) << "Doing " << data.name () << "job";
+
+            //The amount of time that the job was in the queue
+            auto const q_time = date::ceil<microseconds>( 
                 start_time - job.queue_time());
-            perfLog_.jobStart(type, us, start_time, instance);
-            if (us >= 10ms)
-                getJobTypeData(type).dequeue.notify(us);
+            perfLog_.jobStart(type, q_time, start_time, instance);
+            
             job.doJob ();
+
+            //The amount of time it took to dequeue and execute the job
+            auto const x_time = date::ceil<microseconds>(
+                Job::clock_type::now() - job.queue_time());
```

Comment:
`job.queue_time()` should be `start_time` and the comment should be updated to reflect the fact that this represents the execution and not the wait time.

Reply:
Thanks. I made the changes.